### PR TITLE
Add code for linear extrapolation plot

### PR DESCRIPTION
**Pull request by royaurko**
I added two functions in cleverhans/utils.py:
 1. get_logits_over_interval() to get the logits over a bounded interval
 2. linear_extrapolation_plot() to plot these logits

Also added the file linear_extrapolation_plot_example.py in the example directory.

### CODE REVIEW

**Comment by royaurko** [comment]
Fixed the commit that was causing the CLA complaint.
